Source/Plugins/bsfOpenAudio/BsOAAudio.cpp

```diff
@@ -12,10 +12,10 @@
 namespace bs
 {
 	OAAudio::OAAudio()
-		:mVolume(1.0f), mIsPaused(false)
+		:mVolume(1.0f), mIsPaused(false), mDevice(nullptr)
```

Comment:
Could you do this as default values in the class declaration? That would allow the constructor to be marked as "= default;"

Reply:
Oh I agree that making those into default values in class declaration could be done. But from what I've seen of the bsf internals, this particular c++11 feature is mostly used for structs but not for classes ? I've tried to maintain consistency with the rest of bsf codebase.

Reply:
Ultimately I'd like to move all code to use in-class initializers. Constructor initializer lists were the old way of doing things. Note that I would prefer not to use `= default` constructor if the constructor itself contains no code and there are no other constructors. Preferably we just get rid of the constructor altogether.